src/atmos_param/two_stream_gray_rad/two_stream_gray_rad.F90

```diff
@@ -223,6 +243,18 @@ subroutine two_stream_gray_rad_init(is, ie, js, je, num_levels, axes, Time, lonb
 else if(uppercase(trim(rad_scheme)) == 'SCHNEIDER') then
   lw_scheme = B_SCHNEIDER_LIU
   call error_mesg('two_stream_gray_rad','Using Schneider & Liu (2009) radiation scheme for GIANT PLANETS.', NOTE)
+<<<<<<< HEAD
```

Comment:
Is this in the final version you're committing?

Reply:
Thanks for catching this @jamesp ! On one of the times I got a merge conflict, my text editor (which catches finds them) decided to malfunction, so I had to do it manually! It seems I've missed some, so I'll look to tidy it up.

Reply:
Removed the artefact from merge commit now.